core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -938,6 +956,12 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey CLIENT_CACHE_STATE =
+      new Builder(Name.CLIENT_CACHE_STATE)
+          .setDescription("State of the cache: 0 (NOT_IN_USE), 1 (READ_ONLY) and 2 (READ_WRITE)")
```

Comment:
Should this be -1, 0, and 1?

Reply:
I feel 0,1,2 is more friendly for monitoring? my would go with 0, 1, 2

Reply:
Yeah I agree, but is that how the value works in the code, doesn't the Metric.STATE counter initialize at 0?

Reply:
ah you are right. Let me update the state transistion.